### PR TITLE
LPS-122888 when update file from OS if the filename change the title must be the filename without the extension.

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
Just started reviewing :)

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/webdav/DLWebDAVStorageImpl.java [hunk 1]
```diff
@@ -1220,6 +1220,19 @@ protected Resource toResource(
...
 
+	private FileEntry _getFileEntryByName(
+			long groupId, long parentFolderId, String name)
+		throws PortalException {
+
+		try {
+			return _dlAppService.getFileEntry(groupId, parentFolderId, name);
+		}
+		catch (NoSuchFileEntryException noSuchFileEntryException) {
+			return (FileEntry)DLFileEntryUtil.findByG_F_FN(
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/webdav/DLWebDAVStorageImpl.java]: Is there any reason you're calling the persistence directly? Here we should be calling DLFileEntry*Service methods only.

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/webdav/DLWebDAVStorageImpl.java [hunk 2]
```diff
@@ -1026,6 +1027,11 @@ protected String getContentType(
 		return resources;
 	}
 
+	protected String getFileName(String[] pathArray) {
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/webdav/DLWebDAVStorageImpl.java:1036]: Can we make this method private?

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/webdav/DLWebDAVStorageImpl.java (hunk 1 above)
Comment (adolfopa) [reply modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/webdav/DLWebDAVStorageImpl.java]: Nevermind, I see you change this later.

File: portal-impl/src/com/liferay/portlet/documentlibrary/webdav/DLFileEntryResourceImpl.java [hunk 3]
```diff
@@ -118,7 +118,7 @@ public boolean isLocked() {
 
 	private static String _getName(FileEntry fileEntry, boolean appendPath) {
 		if (appendPath) {
-			String name = fileEntry.getTitle();
+			String name = fileEntry.getFileName();
```
Comment (adolfopa) [inline portal-impl/src/com/liferay/portlet/documentlibrary/webdav/DLFileEntryResourceImpl.java:121]: Can you inline this variable?

**Comment by adolfopa** [comment]
Other than the minor things I've mentioned in the review, this looks good 👍

File: modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/webdav/DLWebDAVStorageImpl.java (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/webdav/DLWebDAVStorageImpl.java]: There is no reason.
In this point this is totally wrong.
Thanks

**Comment by AliciaGarciaGarcia** [comment]
ci:stop

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**